src/main/java/frc/robot/subsystems/arm/Arm.java

```diff
@@ -228,5 +233,9 @@ public void updateInputs() {
         loggerInputs.elbowMotorPower = elbowMainMotor.getMotorOutputPercent();
         loggerInputs.shoulderSetpoint = shoulderSetpoint;
         loggerInputs.shoulderSetpoint = elbowSetpoint;
+        loggerInputs.shoulderMotorOffset = 90;
+        loggerInputs.elbowMotorOffset = 180;
```

Comment:
Why are these constants?

Reply:
I deleted them, we decided to use a way of resetting the arm where the offset doesn't really matter